experimental/rules_python/wheelmaker.py

```diff
@@ -101,6 +101,11 @@ def arcname_from(name):
 
             return normalized_arcname
 
+        # If anywhere in the dependency tree a directory is set
+        # it will be passed in as an input file and we need to
+        # ignore.
+        if os.path.isdir(real_filename):
+            return
```

Comment:
shouldn't there be a TODO here to actually copy in the directory contents? this PR is just a temp workaround right?
"we need to ignore" seems surprising from a user perspective

Reply:
I can't see a case where people will want an empty directory as part of their build. They may have another step that populates it but that case is already taken care of